ktor-swagger-ui-examples/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ version = projectVersion
 
 plugins {
     kotlin("jvm")
-    kotlin("plugin.serialization") version "1.9.25"
+    kotlin("plugin.serialization") version "2.0.20"
```

Comment:
serialization's version also could be update to `2.0.21` 😃

Reply:
thank you for finding it, already bumped it